source/common/http/filter_manager.cc

```diff
@@ -812,7 +812,16 @@ void FilterManager::sendLocalReplyViaFilterChain(
   Utility::sendLocalReply(
       state_.destroyed_,
       Utility::EncodeFunctions{
-          modify_headers,
+          [this, modify_headers](ResponseHeaderMap& headers) -> void {
+            if (streamInfo().route_entry_ &&
```

Comment:
Is it possible to make a test for the case where `route_entry_` is nullptr?

Reply:
every early response test (invalid HTTP/1.1 sending 400 before we have headers and can select route) tests this.  Did you want a unit test as well, or just to make sure we had coverage?

Reply:
Just wanted to make sure we have coverage